hknweb/academics/serializers/logistics_serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from ..models.logistics import Course, Department, Instructor, Semester
+
+
+class CourseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Course
+        fields = ['id']
```

Comment:
Same kind of question as above, is there a reason you didn't include the name of the course in the Course model?

Reply:
Analogous response as above.